### DISCUSSION TITLE
TubesZB's new 2022 design of his PoE (Power-over-Ethernet) CC2652 Zigbee Coordinator is out!

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#945] [General]
TubesZB's new 2022 design of his PoE is modular so should also make it possible for him to make models with different Zigbee chips?

Quoting @tube0013

"_Does the same as the old, with a new smaller case/package. Modular design to easily swap out the Olimex ESP32-PoE board, or Soon pop the zigbee module into a companion serial-USB board_."

https://twitter.com/TubesZb/status/1509596528205254660

https://www.tubeszb.com/product/cc2652_poe_coordinator/47

https://community.home-assistant.io/t/tubes-zb-coordinators-and-routers-was-zigbee-router-on-steroids/280896/613

![image](https://user-images.githubusercontent.com/6320001/161271240-7a59eacc-85d4-4e92-b9ec-78523d51d71d.png)

![image](https://user-images.githubusercontent.com/6320001/161271356-2118bd4c-bd47-4165-81aa-435a6ba3d9bc.png)

![image](https://user-images.githubusercontent.com/6320001/161271394-e3621245-d639-46bf-8457-54eabdbe5dcd.png)

**Comment by Hedda** [comment]
ZigStar LAN Gateway by @mercenaruss also uses a modular design stacking the Zigbee radio board above his non-PoE ESP32 LAN board:

https://zig-star.com/projects/zigbee-gw-lan/

https://github.com/mercenaruss/zigstar_gateways

A big difference there is that his radio board implement a USB (USB-C) port as standard

**Comment by tube0013** [comment]
<img width="194" alt="image" src="https://user-images.githubusercontent.com/11084412/161298615-ba822b21-e23b-4ad9-8603-0a99629b70e0.png">

**Reply by Hedda** [reply]
@tube0013  I was more thinking that the USB converter/bridge be on the same board as the Zigbee radio, two-board sandwich-style similar to the ZigStar LAN Gateway. The downside to that is would be more expensive and a little less modular as the USB converter/bridge would always be included:

![image](https://user-images.githubusercontent.com/6320001/161710198-4698f156-1a0f-4294-8f1d-eb1a4a35dc09.png)

![image](https://user-images.githubusercontent.com/6320001/161710105-fff39c5e-5a1f-4596-936d-5d7d7f47a8e1.png)

![image](https://user-images.githubusercontent.com/6320001/161710239-a2381667-b98f-4557-bf25-5d54c5877de9.png)

![image](https://user-images.githubusercontent.com/6320001/161710389-4aaa75d7-5f2d-416b-9fb6-00ddcbfce0e6.png)

https://zig-star.com/projects/zigbee-gw-lan/

https://github.com/mercenaruss/zigstar_gateways
